packages/playground/src/calculator/resource_pricing.vue

```diff
@@ -2,9 +2,14 @@
   <v-container class="custom-container">
     <v-card color="title" class="d-flex justify-center items-center mt-3 pa-3 text-center">
       <v-icon size="30" class="pr-3" color="white">mdi-calculator</v-icon>
-      <v-card-title class="pa-0" lor="white">Resource Pricing Calculator</v-card-title>
+      <v-card-title class="pa-0" lor="white">Pricing Calculator</v-card-title>
```

Comment:
```suggestion
      <v-card-title class="pa-0">Pricing Calculator</v-card-title>
```
All good! Just a small typo that needs to be removed.